connection.go

```diff
@@ -377,6 +365,9 @@ func Connect(addr string, opts Opts) (conn *Connection, err error) {
 	if conn.opts.Concurrency == 0 || conn.opts.Concurrency > maxprocs*128 {
 		conn.opts.Concurrency = maxprocs * 4
 	}
+	if conn.opts.Dialer == nil {
+		conn.opts.Dialer = TntDialer{}
```

Comment:
As far as I know, our current policy is to use `tt` as a shortcut for `tarantool`, not `tnt`.

Reply:
Renamed: `TntDialer` -> `TtDialer`.